python/feathub/online_stores/memory_online_store.py

```diff
@@ -138,3 +175,13 @@ def get(
         if table_info.timestamp_field is not None and not include_timestamp_field:
             features = features.drop(columns=[field_to_drop])
         return features
+
+    def reset(self) -> None:
+        self.table_infos = {}
+
+    @staticmethod
+    def get_instance() -> MemoryOnlineStore:
+        if MemoryOnlineStore.INSTANCE is None:
+            MemoryOnlineStore.INSTANCE = MemoryOnlineStore()
+
+        return MemoryOnlineStore.INSTANCE
```

Comment:
In the current implementation, Feathub would create a `MemoryOnlineStore` every time this method is invoked, while this PR proposes to create a singleton instance. Would this affect the correctness of feature values when there are multiple feature services or processors accessing the same table name in the memory store?

Reply:
MemoryOnlineStore is only needed in tests. We agree there is no correctness issue in the current design.